src/exifutils/exiftoolworker.py

```diff
@@ -145,6 +145,10 @@ def get_aperture(self) -> str:
         aperture = self.get_tag_with_log("Aperture")
         return f"f/{aperture}" if aperture else "Unknown Aperture"
     
+    def get_dof_in_35mm(self) -> str:
+        '''Not implemented yet, see PillowWorker'''
+        return "Unknown Depth of Field in 35mm format"
```

Comment:
You should be able to get `focal_length`, `focal_length_35mm` and `aperture` using:
```python
focal_length = None
focal_length_35mm = None
aperture = None

if (focal_length := self.get_tag_with_log("FocalLength")) is not None:
  focal_length = float(focal_length)
if (focal_length_35mm := self.get_tag_with_log("FocalLengthIn35mmFormat")) is not None:
  focal_length_35mm = float(focal_length_35mm)
elif (focal_length_35mm := self.get_tag_with_log("FocalLength35efl")) is not None:
  focal_length_35mm = float(focal_length_35mm)
if (aperture := self.get_tag_with_log("Aperture")) is not None:
  aperture = float(aperture)
```

Reply:
I'm trying to give it a shot in a new commit, with similar style in PillowWorker though.